src/components/AlertBanner.js

```diff
@@ -9,30 +9,52 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
+/* There are four severity levels each has its own icon built-in
+ *  error   - red
+ *  warning - orange (** this is what you should probably use)
+ *  info    - blue
+ *  success - green
+ */
+/* The "-05:00" at the end of the dates adjust from EST to UTC
+ * NOTE: DST begins on 3/14/2021 when the offset becomes "-04:00"
+ * NOTE: EST begins on 11/7/2021 when the offset becomes "-05:00"
+ */
```

Comment:
The JS `Date()` API is so dumb. It is tempting to suggest the user should not be specifying the timezone and should simply specify a date in Eastern time, but I agree with your implicit conclusion that that is just way too much effort for this. It just makes me upset.

Reply:
We want our times to be Eastern Time, but then we have to deal with what timezone the server is in.

I thought about trying to solve the timezone problem and I **would** if we were crashing a rover on Mars.  (I'll bet that Mars doesn't even have timezones _yet_).

Reply:
It does not have zones _per se_ insofar as there are not geographical bounds within which rovers (or people) crossing them change time standards in an agreed-upon fashion, but there are many different time standards that are used on Mars.

See https://en.wikipedia.org/wiki/Timekeeping_on_Mars#Summary for a summary of the time standards used by the 7 US landers, which are variants of Local Mean Solar Time and Local True Solar Time. And then you can read the entire article for more time zones used with-respect-to (but not actually on) Mars. It seems somewhat US centric, I don't know what _Hope_ and _Tianwen-1_ use?